RecipeManager-base/src/main/java/haveric/recipeManager/Files.java

```diff
@@ -739,7 +739,7 @@ private void createNameIndex(boolean overwrite) {
         s.append(NL);
         s.append(NL).append("NOTE: The duration is compensated when setting potions in flags, so when using 2 seconds it will last 2 seconds regardless of effect type.");
         s.append(NL);
-        s.append(NL).append("More about potions, effects and custom effects: <a href=\"https://minecraft.gamepedia.com/Status_effect\">https://minecraft.gamepedia.com/Status_effect</a>");
+        s.append(NL).append("More about potions, effects and custom effects: <a href=\"https://minecraft.wiki/w/Status_effect\">https://minecraft.wiki/w/Status_effect</a>");
```

Comment:
Can you update the Status Effect references to just use https://minecraft.wiki/w/Effect? We probably don't need to have it point to a redirect anymore.

Reply:
On it!